commit 8da2e33e0a7f7548db75dc53f6bbf2b62c04faab
Author: Paul Eggert <eggert@cs.ucla.edu>
Date:   Mon Oct 18 15:40:34 2021 -0700

    Palestine falls back October 28 24:00
    
    * NEWS: Mention this (thanks to P Chan).
    * asia (Palestine): Predict future transitions to be
    Oct lastThu 24:00, not Oct Sat>=24 1:00.

diff --git a/NEWS b/NEWS
--- a/NEWS
+++ b/NEWS
@@ -4,6 +4,9 @@ Unreleased, experimental changes
 
   Changes to future timestamps

+    Palestine will fall back 10-28 at 24:00, not 10-30 at 01:00.
+    (Thanks to P Chan.)
+
     Fiji will suspend observance of DST for the 2021/2022 season.  Assume for
     now that it will return next year.  (Thanks to Jashneel Kumar and P Chan.)
 
diff --git a/asia b/asia
--- a/asia
+++ b/asia
@@ -3386,11 +3386,6 @@ Zone	Asia/Karachi	4:28:12 -	LMT	1907
 # shall [end] on Oct 24th 2020 at 01:00AM by delaying the clock by 60 minutes.
 # http://www.palestinecabinet.gov.ps/portal/Meeting/Details/51584
 
-# From Tim Parenti (2020-10-20):
-# Predict future fall transitions at 01:00 on the Saturday preceding October's
-# last Sunday (i.e., Sat>=24).  This is consistent with our predictions since
-# 2016, although the time of the change differed slightly in 2019.
-
 # From Pierre Cashon (2020-10-20):
 # The summer time this year started on March 28 at 00:00.
 # https://wafa.ps/ar_page.aspx?id=GveQNZa872839351758aGveQNZ
@@ -3403,6 +3398,13 @@ Zone	Asia/Karachi	4:28:12 -	LMT	1907
 # For now, guess spring-ahead transitions are at 00:00 on the Saturday
 # preceding March's last Sunday (i.e., Sat>=24).
 
+# From P Chan (2021-10-18):
+# http://wafa.ps/Pages/Details/34701
+# Palestine winter time will start from midnight 2021-10-29 (Thursday-Friday).
+#
+# From Paul Eggert (2021-10-18):
+# Guess future fall transitions on October's last Thursday at 24:00.
+
 # Rule	NAME	FROM	TO	-	IN	ON	AT	SAVE	LETTER/S
 Rule EgyptAsia	1957	only	-	May	10	0:00	1:00	S
 Rule EgyptAsia	1957	1958	-	Oct	 1	0:00	0	-
@@ -3438,7 +3440,8 @@ Rule Palestine	2016	2018	-	Oct	Sat>=24	1:00	0	-
 Rule Palestine	2019	only	-	Mar	29	0:00	1:00	S
 Rule Palestine	2019	only	-	Oct	Sat>=24	0:00	0	-
 Rule Palestine	2020	max	-	Mar	Sat>=24	0:00	1:00	S
-Rule Palestine	2020	max	-	Oct	Sat>=24	1:00	0	-
+Rule Palestine	2020	only	-	Oct	24	1:00	0	-
+Rule Palestine	2021	max	-	Oct	lastThu	24:00	0	-
 
 # Zone	NAME		STDOFF	RULES	FORMAT	[UNTIL]
 Zone	Asia/Gaza	2:17:52	-	LMT	1900 Oct
commit d86edaf1247274a76ae8134256a4aae0e2f5b1c3
Author: Paul Eggert <eggert@cs.ucla.edu>
Date:   Wed Oct 20 09:11:41 2021 -0700

    Palestine falls back October 29 01:00.
    
    * NEWS: Mention this (thanks to Heba Hamad).
    * asia (Palestine): Change from lastThu 24:00 to lastFri 1:00.

diff --git a/NEWS b/NEWS
--- a/NEWS
+++ b/NEWS
@@ -4,8 +4,8 @@ Unreleased, experimental changes
 
   Changes to future timestamps
 
-    Palestine will fall back 10-28 at 24:00, not 10-30 at 01:00.
-    (Thanks to P Chan.)
+    Palestine will fall back 10-29 (not 10-30) at 01:00.
+    (Thanks to P Chan and Heba Hemad.)
 
     Fiji will suspend observance of DST for the 2021/2022 season.  Assume for
     now that it will return next year.  (Thanks to Jashneel Kumar and P Chan.)
diff --git a/asia b/asia
--- a/asia
+++ b/asia
@@ -3402,8 +3402,12 @@ Zone	Asia/Karachi	4:28:12 -	LMT	1907
 # http://wafa.ps/Pages/Details/34701
 # Palestine winter time will start from midnight 2021-10-29 (Thursday-Friday).
 #
-# From Paul Eggert (2021-10-18):
-# Guess future fall transitions on October's last Thursday at 24:00.
+# From Heba Hemad, Palestine Ministry of Telecom & IT (2021-10-20):
+# ... winter time will begin in Palestine from Friday 10-29, 01:00 AM
+# by 60 minutes backwards.
+#
+# From Paul Eggert (2021-10-20):
+# Guess future fall transitions on October's last Friday at 01:00.
 
 # Rule	NAME	FROM	TO	-	IN	ON	AT	SAVE	LETTER/S
 Rule EgyptAsia	1957	only	-	May	10	0:00	1:00	S
@@ -3441,7 +3445,7 @@ Rule Palestine	2019	only	-	Mar	29	0:00	1:00	S
 Rule Palestine	2019	only	-	Oct	Sat>=24	0:00	0	-
 Rule Palestine	2020	max	-	Mar	Sat>=24	0:00	1:00	S
 Rule Palestine	2020	only	-	Oct	24	1:00	0	-
-Rule Palestine	2021	max	-	Oct	lastThu	24:00	0	-
+Rule Palestine	2021	max	-	Oct	lastFri	1:00	0	-
 
 # Zone	NAME		STDOFF	RULES	FORMAT	[UNTIL]
 Zone	Asia/Gaza	2:17:52	-	LMT	1900 Oct
